src/internal/images/pull.go

```diff
@@ -87,8 +88,11 @@ func PullAll(buildImageList []string, imageTarballPath string) map[name.Tag]v1.I
 		case update.Error != nil && errors.Is(update.Error, io.EOF):
 			progressBar.Success("Pulling %d images (%s)", len(imageMap), utils.ByteFormat(float64(update.Total), 2))
 			return tagToImage
+		case update.Error != nil && strings.HasPrefix(update.Error.Error(), "archive/tar: missed writing "):
+			// Handle potential image cache corruption with a more helpful error. See L#54 in libexec/src/archive/tar/writer.go
+			message.Fatalf(update.Error, "potential image cache corruption: %s of %v bytes", update.Error.Error(), update.Total)
```

Comment:
Does it make sens to plug the clear cache command here as a possible fix action?

Reply:
Yes I think we should either mention the command or at least a link to a doc for how to fix it as the message is a bit cryptic otherwise.  Keep in mind most of the time this will be on an internet-connected system when we see this error message.  

Down the road when we embed the docusarus site into the web ui we could just link to that even for airgap.

Reply:
How about this:
![image](https://user-images.githubusercontent.com/3977569/194408726-ed405d62-34a3-4584-bf48-c5f78520eb8e.png)
Didn't link to docs since that seemed brittle (and the docs page isn't actually there yet)